cli/command/formatter/history_test.go

```diff
@@ -50,6 +50,10 @@ func TestHistoryContext_ID(t *testing.T) {
 }
 
 func TestHistoryContext_CreatedSince(t *testing.T) {
+	dateStr := "2009-11-10T23:00:00Z"
+	if runtime.GOOS == "windows" {
+		dateStr = "2009-11-11T00:00:00+01:00"
```

Comment:
arf

Is this dependent on system configuration?

Reply:
I'm actually not sure 😓 cc @simonferquel @johnstep

Reply:
It seems. the good thing is that the timezone is in the string here, and that the format is in a standard parsable from the time package. Comparison should be done after date/time parsing instead that on strings would be better though (and would not require windows specific code).

Reply:
Since we're testing a formatter, which is responsible for formatting output that is seen by a user, I think we should be comparing the string, not a `time.Time`. The format is relevant here.